CppImport/src/CppImportManager.cpp

```diff
@@ -78,6 +78,9 @@ Model::TreeManager*CppImportManager::createTreeManager(const bool statisticsPerP
 		SAFE_DELETE(frontendActionFactory);
 		SAFE_DELETE(tool);
 	}
+
+	visitor->macroImportHelper_.finalize();
```

Comment:
`finalize` ? finalize what? This reads a bit like java. Give it a more descriptive name. I'm not sure what this does exactly, but perhaps `finalizeGeneration` or `finalizeMacrosInScope` or something else is better.